Moodis/Moodis/Feature/Statistics/CalendarForm.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Windows.Forms;
+using System.Collections.Generic;
+using Moodis.Ui;
+using Moodis.Feature.Login;
+using System.Linq;
+
+namespace Moodis.Feature.Statistics
+{
+    public partial class CalendarForm : Form
+    {
+        private CalendarViewModel calendarViewModel;
+        private Form parentForm;
+        private const string FormatDouble = "N3";
+
+        public CalendarForm(CalendarViewModel viewModel, Form form)
+        {
+            InitializeComponent();
+            calendarViewModel = viewModel;
+            parentForm = form;
+            updateView();
+        }
+
+        private void SelectDate(object sender, DateRangeEventArgs e)
+        {
+            updateView();
+        }
+
+        private void FormClose(object sender, FormClosedEventArgs e)
+        {
+            parentForm.Show();
+            Dispose();
+        }
+
+        private void updateView()
+        {
+            listOfData.Items.Clear();
+            calendarViewModel.updateViewModel();
+
+            int counter = 0, counterM = 0, counterD = 0;
```

Comment:
var

Reply:
Can't change to var since declaring multiple variables.